pipelines/tekton-bundle-builder/kustomization.yaml

```diff
@@ -27,6 +27,8 @@ patches:
       path: /spec/tasks/9  # sbom-json-check
     - op: remove
       path: /spec/tasks/5  # deprecated-base-image-check
+    - op: remove
+      path: /spec/tasks/4  # build-source-image
```

Comment:
@tkdchen, I added this removal for the source image tasks which I think was missed in https://github.com/redhat-appstudio/build-definitions/pull/621